diff --git a/files/debs/nova b/files/debs/nova
index 58dad41..85183df 100644
--- a/files/debs/nova
+++ b/files/debs/nova
@@ -10,14 +10,10 @@ iputils-arping
 kpartx
 libjs-jquery-tablesorter # Needed for coverage html reports
 libmysqlclient-dev
-libvirt-bin # NOPRIME
-libvirt-dev # NOPRIME
 mysql-server # NOPRIME
 parted
 pm-utils
 python-mysqldb
-qemu # dist:wheezy,jessie NOPRIME
-qemu-kvm # NOPRIME
 rabbitmq-server # NOPRIME
 socat # used by ajaxterm
 sqlite3
diff --git a/lib/nova_plugins/functions-libvirt b/lib/nova_plugins/functions-libvirt
index 6b7c7c2..5a48761 100644
--- a/lib/nova_plugins/functions-libvirt
+++ b/lib/nova_plugins/functions-libvirt
@@ -24,12 +24,15 @@ DEBUG_LIBVIRT=$(trueorfalse True DEBUG_LIBVIRT)
 function install_libvirt {
     if is_ubuntu; then
         if is_arch "aarch64" && [[ ${DISTRO} == "trusty" ]]; then
-            install_package qemu-system
+            # install_package qemu-system
+            echo "mz: ignore package qemu-system"
         else
-            install_package qemu-kvm
+            # install_package qemu-kvm
+            echo "mz: ignore package qemu-kvm"
             install_package libguestfs0
         fi
-        install_package libvirt-bin libvirt-dev
+        # install_package libvirt-bin libvirt-dev
+        echo "mz: ignore packages libvirt-bin and libvirt-dev"
         pip_install_gr libvirt-python
         if [[ "$EBTABLES_RACE_FIX" == "True" ]]; then
             # Work around for bug #1501558. We can remove this once we
@@ -124,13 +127,15 @@ EOF
 
     # Service needs to be started on redhat/fedora -- do a restart for
     # sanity after fiddling the config.
-    restart_service $LIBVIRT_DAEMON
+    # restart_service $LIBVIRT_DAEMON
+    echo "mz: ignore restart of libvirt daemon"
 
     # Restart virtlogd companion service to ensure it is running properly
     #  https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1577455
     #  https://bugzilla.redhat.com/show_bug.cgi?id=1290357
     # (not all platforms have it; libvirt 1.3+ only, thus the ignore)
-    restart_service virtlogd || true
+    # restart_service virtlogd || true
+    echo "mz: ignore restart of virtlogd daemon"
 }
 
 
